packaging/docker/docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       - BEE_DB_DISABLE_SEEKS_COMPACTION
       - BEE_DEBUG_API_ADDR
       - BEE_DEBUG_API_ENABLE
+      - BEE_FULL_NODE
```

Comment:
Please remove this line, then your PR is good to go!

Reply:
this line is necessary if we need to run node in full mode by setting `BEE_FULL_NODE` in env file.

Reply:
You are right!